general_rules/Organization.tex

```diff
@@ -23,31 +23,17 @@ \subsection{Stage System}\label{rule:stages}
 Note that the same task can be performed multiple times during \SONE{} and \STWO{} (see~\refsec{rule:score_system}).
 In case of having no considerable score deviation between a team advancing to the next stage and a team dropping out, the TC may announce additional teams advancing to the next stage.
 
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection{Thematic Scenarios}
-\label{rule:themes}
-
-Each stage includes a set of tasks that are grouped in two thematic scenarios:
-\begin{enumerate}
-	\item The \iterm{Housekeeper} scenario features tasks related to cleaning, organization, and maintenance of a household.
-	\item The \iterm{Party Host} scenario focuses on providing general assistance during a party by attending to the needs of the guests.
-\end{enumerate}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Schedule}
 \label{rule:schedule}
 
 \begin{enumerate}
-	\item \textbf{Thematic scenario blocks:} Two \Testblocks{} are scheduled per day, lasting between two and three hours.
-	All teams have at least 2 \Testslots{} assigned per \Testblock{}, during which they can perform any task of their choice from the block's assigned scenario.
+	\item \textbf{Test Blocks:} For each test one Test Block is scheduled during which each (qualified) team gets at least two Test Slots assigned. With the Restaurant test being the exception where only one Test Slot is assigned.
```

Comment:
#TODO add what was discussed last TC meeting. The GPSR slot after stage1 can be used to re-take other tests.

Reply:
Added